.babelrc

```diff
@@ -1,3 +1,10 @@
 {
-  "presets": ["es2015", "stage-0", "react"]
+  "presets": [
+    ["es2015", {"modules": false}],
```

Comment:
@Kikobeats disable modules has no effect on hot loader.  Webpack now can handle es 6 modules and for better tree shaking this has to be off in babel.